tests/customer-resource-show-test.js

```diff
@@ -40,74 +41,78 @@ describeApplication('CustomerResourceShow', () => {
       this.server.create('contributor', { type: 'illustrator', contributor: 'Artist, John' })
     ].map(m => m.toJSON());
 
+    title.subjects = [
+      this.server.create('subject', { type: 'TLI', subject: 'Writing' }),
+      this.server.create('subject', { type: 'TLI', subject: 'Words' }),
+    ].map(m => m.toJSON());
+
     title.save();
 
     resource = this.server.create('customer-resource', {
       package: providerPackage,
       isSelected: false,
-      title
+      title,
+      url: 'frontside.io'
     });
   });
 
   describe('visiting the customer resource page', () => {
     beforeEach(function () {
       return this.visit(`/eholdings/customer-resources/${resource.titleId}`, () => {
-        expect(ResourcePage.$root).to.exist;
+        expect(ResourcePage).to.exist;
```

Comment:
I now realize that there isn't any docs around this on the `@bigtest/interaction` repo, but `ResourcePage.$root` is still the correct way to verify that the root element of the resource page exists.

Also `.to.exist` does not work as you may think here. `chai-jquery` [overrides the built-in chai assertion](https://github.com/chaijs/chai-jquery#exist), and if it isn't a jQuery element, it will call the original implementation instead.

The original implementation of `.to.exist` should work for the bigtest page object because it will throw an error when `.$root` doesn't exist. But for other jQuery matchers, we'll want to use something like [chai-dom](https://github.com/nathanboktae/chai-dom) instead.

Reply:
Cool, went back to checking for `ResourcePage.$root` to exist.